packages/kbn-es-query/tsconfig.json

```diff
@@ -1,7 +1,12 @@
 {
   "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./target",
+    "target": "ES3"
```

Comment:
@elastic/kibana-platform I had to set the `target` to fix the error in CI [here](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/8080/JOB=kibana-ciGroup1,node=immutable/console). Is this the correct way to handle TS compilation in a package? In the [root kibana tsconfig](https://github.com/elastic/kibana/blob/686676c38d4922b2db5fb80763e5f50cbc7ff1f0/tsconfig.json#L25-L27) `target` is set to `esnext` and there's a comment about it being overridden in webpack, but that doesn't seem to happen for packages maybe?

Reply:
@Bargs I think you can instead just extend kibana ts config for browser like:

`"extends": "../../tsconfig.browser.json",`

Everything should work then

Reply:
thanks @mistic !